bpf/aya-bpf/src/maps/array.rs

```diff
@@ -47,14 +47,27 @@ impl<T> Array<T> {
         }
     }
 
+    #[inline(always)]
     pub fn get(&self, index: u32) -> Option<&T> {
-        unsafe {
-            let value = bpf_map_lookup_elem(
-                self.def.get() as *mut _,
-                &index as *const _ as *const c_void,
-            );
-            // FIXME: alignment
```

Comment:
Also you removed the FIXME without fixing it :P

Reply:
sorry, for now I will put the comment back

but then I will try to actually fix it